test/micro/org/openjdk/bench/vm/compiler/MergeLoadBench.java

```diff
@@ -317,7 +317,7 @@ public void getCharB(Blackhole BH) {
     public void getCharBV(Blackhole BH) {
         long sum = 0;
         for (int i = 0; i < longs.length; i++) {
-            char c = (char) CHAR_B.get(bytes4, Unsafe.ARRAY_BYTE_BASE_OFFSET + i * 2);
+            char c = (char) CHAR_B.get(bytes4, (int)(Unsafe.ARRAY_BYTE_BASE_OFFSET + i * 2));
```

Comment:
char c = (char) CHAR_B.get(bytes4, i * 2);

it should be changed like this

Reply:
Thanks. The `Unsafe.ARRAY_BYTE_BASE_OFFSET` has been removed.